src/dss/main.py

```diff
@@ -130,5 +134,34 @@ def status_command(kubeconfig: str) -> None:
     get_status(lightkube_client)
 
 
+@main.command(name="list")
+@click.option(
+    "--wide",
+    default=False,
+    is_flag=True,
+    help="Display full information without truncation.",
+)
+@click.option(
+    "--kubeconfig",
+    help="Path to a Kubernetes config file. Defaults to the value of the KUBECONFIG environment variable, else to './kubeconfig'.",  # noqa E501
+)
+def list_command(kubeconfig: str, wide: bool):
+    """
+    Lists all created notebooks in the DSS environment.
+
+    The output is truncated to 80 characters. Use the --wide flag to display full information.
+    """
+    try:
+        kubeconfig = get_default_kubeconfig(kubeconfig)
+        lightkube_client = get_lightkube_client(kubeconfig)
+        list_notebooks(lightkube_client, wide)
+    except ApiError as e:
+        logger.error(f"Error interacting with Kubernetes: {e}", err=True)
+        sys.exit(1)
+    except Exception as e:
+        logger.error(f"An error occurred: {e}", err=True)
```

Comment:
There's a syntax error here - `logger.error()` does not have an `err` keyword arg.  Repro this by running `dss list` without a kubeconfig